data/schema/core/addons.cfg

```diff
@@ -0,0 +1,470 @@
+
+#define ADDON_DESCRIPTION
+	{SIMPLE_KEY name t_string}
+	{SIMPLE_KEY description t_string}
+#enddef
+
+#define BASE_COMPATIBILITY_KEYS
+	{DEFAULT_KEY allow_new_game bool yes}
+	{SIMPLE_KEY allow_era string_list}
+	{SIMPLE_KEY disallow_era string_list}
+	{SIMPLE_KEY ignore_incompatible_era string_list}
+	{SIMPLE_KEY allow_modification string_list}
+	{SIMPLE_KEY disallow_modification string_list}
+	{SIMPLE_KEY ignore_incompatible_modification string_list}
+#enddef
+
+[tag]
+	name="$side"
+	max=0
+	# TODO: This superclass should only be used if no_leader=no
+	super="units/$unit"
+	{DEFAULT_KEY no_leader bool no}
+	{SIMPLE_KEY recruit string}
+	{DEFAULT_KEY gold int 100}
+	{DEFAULT_KEY income int 0}
+	{DEFAULT_KEY hidden bool no}
+	{SIMPLE_KEY fog bool}
+	{SIMPLE_KEY shroud bool}
+	{SIMPLE_KEY fog_data string}
+	{SIMPLE_KEY shroud_data string}
+	{SIMPLE_KEY persistent bool}
+	{SIMPLE_KEY save_id string}
+	{SIMPLE_KEY previous_save_id string}
+	{SIMPLE_KEY team_name string}
+	{SIMPLE_KEY user_team_name t_string}
+	{SIMPLE_KEY current_player string}
+	{SIMPLE_KEY side_name t_string}
+	{SIMPLE_KEY color string}
+	{SIMPLE_KEY flag string}
+	{SIMPLE_KEY flag_icon string}
+	{SIMPLE_KEY village_gold int}
+	{SIMPLE_KEY village_support int}
+	{SIMPLE_KEY recall_cost int}
+	{DEPRECATED_KEY share_maps string}
+	{DEPRECATED_KEY share_view string}
+	{SIMPLE_KEY share_vision share_vision}
+	{DEFAULT_KEY scroll_to_leader bool yes}
+	{DEFAULT_KEY suppress_end_turn_confirmation bool no}
+	{DEFAULT_KEY defeat_condition defeat_condition no_leader_left}
+	[tag]
+		name="unit"
+		max=infinite
+		super="units/$unit"
+	[/tag]
+	[tag]
+		name="leader"
+		max=infinite
+		super="units/$unit"
+	[/tag]
+	[tag]
+		name="village"
+		max=infinite
+		{SIMPLE_KEY x int}
+		{SIMPLE_KEY y int}
+	[/tag]
+	[tag]
+		name="fog_override"
+		max=infinite
+		{SIMPLE_KEY x int_list}
+		{SIMPLE_KEY y int_list}
+	[/tag]
+	[tag]
+		name="ai"
+		max=infinite
+		super="$ai"
+	[/tag]
+[/tag]
+
+[tag]
+	name="multiplayer_side"
+	max=infinite
+	super="$side"
+	{REQUIRED_KEY id string}
+	{SIMPLE_KEY name t_string}
+	{SIMPLE_KEY description t_string}
+	{SIMPLE_KEY image string}
+	{SIMPLE_KEY leader string}
+	{SIMPLE_KEY random_leader string}
+	{DEFAULT_KEY random_faction bool no}
+	# TODO: below two keys only valid if above key is yes
+	{SIMPLE_KEY choices string_list}
+	{SIMPLE_KEY except string_list}
+	{SIMPLE_KEY terrain_liked terrain_list}
+
+	{DEFAULT_KEY allow_player bool yes}
```

Comment:
i'm quite sure this and below are not allowed in [multiplayer faction]

Reply:
I don't recall why I included then there, but... according to the wiki, `[multiplayer_side]` accepts everything that `[side]` does. In fact, it says the former is simply merged into the latter when a game starts.

Reply:
Though thinking further on it, even if it's allowed, `allow_player` at the very least would completely defeat the point of the `[multiplayer_side]` tag.

Reply:
I think what happened here is I cut-and-pasted these from another area but intended to drop them in `[side]`, not `[multiplayer_side]`.